runtime/ftplugin/nroff.vim

```diff
@@ -22,7 +22,19 @@ setlocal comments=:.\\\"
 setlocal sections+=Sh
 setlocal define=.\s*de
 
-let b:undo_ftplugin = 'setlocal commentstring< comments< sections< define<'
+if get(b:, 'preprocs_as_sections')
+  setlocal sections=EQTSPS[\ G1GS
```

Comment:
This overrides the `sections` settings from 2 lines above. Is that intended? Then possible this should be moved to an `else` part

Reply:
Since 'sections' is used for only nroff and `Sh` is always appended, can we update the default value?
https://github.com/vim/vim/blob/f39de60608c5bf2106ad6ce08b8dbe03b271e3ff/src/optiondefs.h#L2218-L2221

Reply:
reverted